src/c++/perf_analyzer/genai-perf/docs/files.md

```diff
@@ -0,0 +1,130 @@
+<!--
+Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+ * Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+ * Neither the name of NVIDIA CORPORATION nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Generated File Structures
+
+## Overview
+
+This document serves as a guide to understanding the structure and contents of
+the files generated  by GenAi-Perf.
+
+## Directory Structure
+
+After running GenAi-Perf, your file tree should contain the following:
+
+```
+genai-perf/
+├── artifacts/
+│   ├── data/
+│   └── images/
+```
+
+## File Types
+Within the artifacts and docs directories, several file types are generated,
+including .gzip, .csv, .json, .html, and .jpeg. Below is a detailed
+explanation of each file and its purpose.
+
+### Artifacts Directory
+
+#### Data Subdirectory
+
+The data subdirectory contains the raw and processed performance data files.
+
+##### GZIP Files
+
+- all_data.gzip: Aggregated performance data from all collected metrics.
```

Comment:
@debermudez Do you know if this is per request? It's also a bit confusing, as I'm seeing datapoint 0 have request_throughputs but no other rows have that. And I don't know how token-to-token times factor in for this chart. Any clarification here coming from you would be great. Feel free to even commit a description directly, if easier.

Also, did I capture per-request versus per-token correctly for the other GZIP files?

Reply:
All of the data in a parquet file was a request from mark moyou.  
If the data is confusing however, we should clean it up. Some of the data only had 1 datapoint while the others have data for every request which led to the discrepancy.

Reply:
There is an offline conversation with Mark and Elias. It sounds like the columns are all concatenated, so the values in the rows have no real relationship with one another. Depending on Mark's needs, we may be able to remove this file. If we do so in a separate PR, we can remove this line.